torchgeo/datasets/dl4gam.py

```diff
@@ -0,0 +1,446 @@
+
+    Given that glacier ice is not always visible in the images, due to seasonal snow,
+    shadow/cloud cover and, most importantly, debris cover, the dataset also includes
+    additional features that can help in the segmentation task.
+
+    Dataset features:
+
+    * Sentinel-2 images (all bands, including cloud and shadow masks which can be used
+      for loss masking)
+    * glacier mask (0: no glacier, 1: glacier)
+    * debris mask (0: no debris, 1: debris) based on a mix of three publications
+      (`Scherler et al. 2018 <https://doi.org/10.5880/GFZ.3.3.2018.005>`_,
+      `Herreid & Pellicciotti 2020 <https://doi.org/10.5281/zenodo.3866466>`_,
+      `Linsbauer et al. 2021
+      <https://doi.glamos.ch/data/inventory/inventory_sgi2016_r2020.html>`_)
+    * DEM (Copernicus GLO-30) + five derived features
+      (using `xDEM <https://github.com/GlacioHack/xdem>`_): slope,
+      aspect, terrain ruggedness index, planform and profile curvatures
+    * dh/dt (surface elevation change) map over 2010-2015
+      (`Hugonnet et al. 2021 <https://doi.org/10.6096/13>`_)
+    * v (surface velocity) map over 2015 (`ITS_LIVE <https://its-live.jpl.nasa.gov/>`_)
+
+    Other specifications:
+
+    * temporal coverage: one acquisition per glacier, from either 2015 (mainly), 2016,
+      or 2017
+    * spatial coverage: only glaciers larger than 0.1 km\ :sup:`2`\  are considered
+      (n=1593, after manual QC), totalling ~1685 km\ :sup:`2`\  which represents ~93% of
+      the total inventory area for this region
+    * 2251 patches sampled with overlap from the 1593 glaciers;
+      or 11440 for the `large` version, obtained with an increased sampling overlap
+    * the dataset download size is 5.8 GB (11 GB when unarchived);
+      or 29.5 GB (52 GB when unarchived) for the `large` version
+    * the dataset is provided at 10m GSD (after bilinearly resampling some of the
+      Sentinel-2 bands and the additional features which come at a lower resolution)
+    * the dataset provides fixed training, validation, and test geographical splits
+      (70-10-20, by glacier area)
+    * five different splits are provided, according to a five-fold cross-validation
+      scheme
+    * all the features/masks are stacked and provided as NetCDF files (one or more per
+      glacier), structured as
+      `data/{glacier_id}/{glacier_id}_{patch_number}_{center_x}_{center_y}.nc`
+    * data is projected and geocoded in local UTM zones
+
+    For more details check also: https://huggingface.co/datasets/dcodrut/dl4gam_alps
+
+    If you use this dataset in your research, please cite the following paper:
+
+    * https://doi.org/10.22541/essoar.173557607.70204641/v1
+
+    .. note::
+
+        This dataset requires the following additional libraries to be installed:
+
+        * `xarray <https://pypi.org/project/xarray/>`_
+        * `netcdf4 <https://pypi.org/project/netCDF4/>`_
+           or `h5netcdf <https://pypi.org/project/h5netcdf/>`_
```

Comment:
```suggestion
          or `h5netcdf <https://pypi.org/project/h5netcdf/>`_
```
Sphinx is really sensitive to indentation: https://torchgeo--2508.org.readthedocs.build/en/2508/api/datasets.html#dl4gam